lib/crawly/manager_api.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Crawly.ManagerAPI do
```

Comment:
I would want to avoid adding extra modules here. Please put add workers code to the manager gen server.

Reply:
ok